godot-core/src/builtin/vectors/vector2i.rs

```diff
@@ -21,7 +21,7 @@ use std::fmt;
 /// required. Note that the values are limited to 32 bits, and unlike [`Vector2`] this cannot be
 /// configured with an engine build option. Use `i64` or [`PackedInt64Array`] if 64-bit values are
 /// needed.
-#[derive(Default, Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Debug)]
+#[derive(Default, Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Debug, Hash)]
```

Comment:
Nitpick: `Hash` should come before `Debug`.

(I still need to write these conventions down somewhere...)